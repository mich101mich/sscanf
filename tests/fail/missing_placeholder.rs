fn main() {
    sscanf::sscanf!("hi", "asdf{}", usize, std::vec::Vec<std::string::String>, u32);
    sscanf::sscanf!("hi", "asdf{u32}{}{usize}", usize, u32);
}
