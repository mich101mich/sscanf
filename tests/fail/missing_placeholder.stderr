error: unused type
 --> tests/fail/missing_placeholder.rs:2:44
  |
2 |     sscanf::sscanf!("hi", "asdf{}", usize, std::vec::Vec<std::string::String>, u32);
  |                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: unused type
 --> tests/fail/missing_placeholder.rs:2:80
  |
2 |     sscanf::sscanf!("hi", "asdf{}", usize, std::vec::Vec<std::string::String>, u32);
  |                                                                                ^^^

error: unused type
 --> tests/fail/missing_placeholder.rs:3:56
  |
3 |     sscanf::sscanf!("hi", "asdf{u32}{}{usize}", usize, u32);
  |                                                        ^^^
